JythonCli.java

```diff
@@ -162,28 +217,72 @@ void runProcess(String[] args) throws IOException, InterruptedException {
 
         cmd.add("org.python:jython-slim:" + jythonVersion);
 
-        Collections.addAll(cmd, args);
+        cmd.addAll(jythonArgs);
 
-        if (debug) {
-            System.err.println("[jython-cli] " + cmd.toString());
-        }
+        printIfDebug(cmd.toString());
 
         ProcessBuilder pb = new ProcessBuilder(cmd);
         pb.inheritIO();
         pb.start().waitFor();
     }
 
     /**
-     * Main {@code jython-cli} (JythonCli.java) program.
+     * Shorthand to print a line of the source jbang block if {@link #debug} is
+     * set.
      *
-     * @param args arguments to the program. The arguments are exactly the same command-line arguments
-     *             Jython itself supports as documented in <a href="https://www.jython.org/jython-old-sites/docs/using/cmdline.html">Jython Command Line</a>
+     * @param lineno source line number
+     * @param line text captured to {@link #tomlText}
+     */
+    void printIfDebug(int lineno, String line) {
+        if (debug) {
+            System.err.printf("%6d :%s\n", lineno, line);
+        }
+    }
+
+    /**
+     * Shorthand to print something if {@link #debug} is set.
+     *
+     * @param text to print
+     */
+    void printIfDebug(String text) {
+        if (debug) {
+            System.err.printf(text);
```

Comment:
"\n" is missing, should be `System.err.printf(text + "\n");

Reply:
Thanks! I probably meant `println`.